Dockerfile-debian.template

```diff
@@ -106,6 +107,7 @@ RUN { \
         echo 'memory_limit=${PHP_MEMORY_LIMIT}'; \
         echo 'upload_max_filesize=${PHP_UPLOAD_LIMIT}'; \
         echo 'post_max_size=${PHP_UPLOAD_LIMIT}'; \
+        echo 'pm.max_children = ${PHP_PM_MAX_CHILDREN}'; \
```

Comment:
FPM parameters don't belong in the php.ini path (PHP_INI_DIR). FPM stuff needs to go into the php-fpm.d/ path.

We don't currently customize anything FPM related in the Dockerfile but it can probably be modeled after the PHP_INI_DIR stuff. Perhaps /usr/local/etc/php-fpm.d/nextcloud.ini or something like that 

As currently coded, at best this parameter gets ignored.